pkg/kubelet/cm/deviceplugin/device_plugin_stub.go

```diff
@@ -143,7 +182,7 @@ func (m *Stub) Update(devs []*pluginapi.Device) {
 
 // Allocate does a mock allocation
 func (m *Stub) Allocate(ctx context.Context, r *pluginapi.AllocateRequest) (*pluginapi.AllocateResponse, error) {
-	log.Printf("Allocate, %+v", r)
+	glog.V(2).Infof("Allocate, %+v", r)
```

Comment:
would it make sense to use a higher log level eg, 4 ?

Reply:
I don't think log levels really matters for tests :)